lib/orbf/rules_engine/builders/spans/spans.rb

```diff
@@ -125,37 +123,44 @@ def prefix(name)
           name.gsub(REGEX, "")
         end
 
-        # last_6_months => 6
-        def time_offset(name)
-          name.match(REGEX)[1]
-        end
-
-        def time_unit(name)
-          name.match(REGEX)[2]
+        def exlusive_offset_for(exclusive, name)
+          if exclusive == "exclusive"
+            1
+          elsif exclusive.nil?
+            0
+          else
+            raise "not supported exclusive mode : #{name}"
```

Comment:
I like the future proofeness of this, but then we might need to call it `not supported modifier` or something.